vm/src/stdlib/collections.rs

```diff
@@ -420,7 +420,12 @@ mod _collections {
                     .maxlen
                     .map(|maxlen| format!(", maxlen={}", maxlen))
                     .unwrap_or_default();
-                format!("deque([{}]{})", elements.into_iter().format(", "), maxlen)
+                format!(
+                    "{}([{}]{})",
+                    zelf.class().slot_name(),
```

Comment:
is this not `zelf.class().name()` but `zelf.class().slot_name()`?

Reply:
First thought `name()` and `slot_name()` showed the same value, but `name()` returning `self.slot_name().rsplit('.').next().unwrap().to_string()` looks more good to me now. Thanks for reviewing!